agent/bench-scripts/pbench-netperf

```diff
@@ -588,6 +588,7 @@ for protocol in `echo $protocols | sed -e s/,/" "/g`; do
 	done
 done
 printf -- "</pre>\n" >>$benchmark_summary_html_file
+pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir
```

Comment:
Add "`end`" argument, and invoke it with the "`beg`" argument between lines 408 and 409.